src/async-components/views/dialogs/security/ImportE2eKeysDialog.tsx

```diff
@@ -108,7 +108,7 @@ export default class ImportE2eKeysDialog extends React.Component<IProps, IState>
                 return MegolmExportEncryption.decryptMegolmKeyFile(arrayBuffer, passphrase);
             })
             .then((keys) => {
-                return this.props.matrixClient.importRoomKeys(JSON.parse(keys));
+                return this.props.matrixClient.getCrypto()?.importRoomKeys(JSON.parse(keys));
```

Comment:
At this point there should be a cryptoAPI, maybe we can just do `getCrypto()!` or maybe we could have a sort of `getSafeCrypto()` new API that would throw a `End-to-end encryption disabled` like the deprecated`importRoomKeys` was doing

Reply:
Good call - fixed in f1fbff92e1